content/docs/command-reference/params/index.md

```diff
@@ -16,62 +16,63 @@ positional arguments:
 
 ## Description
 
-In order to track parameters and hyperparameters associated to machine learning
-experiments in <abbr>DVC projects</abbr>, DVC provides a different type of
-dependencies: _parameters_. They usually have simple names like `epochs`,
-`learning-rate`, `batch_size`, etc.
+<abbr>Parameters</abbr> are a special type of <abbr>dependency</abbr> to track
```

Comment:
I like the previous one better (goes from a user need vs this one is very formal and very technical)

Reply:
also, should we stop expanding the scope for now? :)

Reply:
> stop expanding the scope

I've moved all params changes to https://github.com/iterative/dvc.org/pull/3899.